src/services/rebuttal.service.js

```diff
@@ -0,0 +1,20 @@
+import {authHeader, stringFormat} from '@/helpers';
+import {config} from "@/configurations";
+import axios from "axios";
+
+export const rebuttalService = { getRebuttals };
+
+function getRebuttals(uuid) {
```

Comment:
This function can go inside debate.service.js - rebuttals are just debates which are linked to parent debate

Reply:
done